app/models/tenure.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Tenure
+  include ActiveModel::Model
+
+  attr_accessor :typeCode, :typeDescription
+
+  class << self
+    def find_tenure_type(reference)
+      response = PlatformApis::TenancyInformation::Client.get_tenancy_information_by_property_reference(reference)
+
+      # The API returns a 200 with response "tenancies": [] when no
+      # tenancy agreement reference can be found for a property.
+      # In this case we will return nil if response["tenancies"] is empty
+      build(response["tenancies"].first["tenureType"]) if response["tenancies"].any?
```

Comment:
Hey If I use prop ref `00072708` for example then this would return `"tenure": null` on the API. This might be what we want but just putting this here in case we wanted to return something else

Reply:
yup, that's what we want
will update the comment